modules/ozoneBidAdapter.js

```diff
@@ -77,13 +78,18 @@ export const spec = {
 
     delete ozoneRequest.test; // don't allow test to be set in the config - ONLY use $_GET['pbjs_debug']
     if (bidderRequest.gdprConsent) {
+      utils.logInfo('ADDING GDPR');
```

Comment:
These debug log entries are likely to be confusing to others - please consider removing them or at least annotating with your bidder name so it's more clear what's adding GDPR.